src/source/rest.rs

```diff
@@ -166,24 +166,32 @@ async fn handle_request(mut req: Request<ServerState>) -> tide::Result<Response>
 
     // TODO: consider stuff like charset here?
     let ct: Option<Mime> = req.content_type();
+    let codec_override = ct.map(|ct| ct.essence().to_string());
 
     // request metadata
     // TODO namespace these better?
-    let mut meta = Value::object_with_capacity(5);
+    let mut meta = Value::object_with_capacity(4);
     meta.insert("request_method", req.method().to_string())?;
```

Comment:
would it make more sense to nest this? instead of `request_...` have `{"requiest": {"...":""}}`